niceday_client/niceday_client.py

```diff
@@ -268,3 +268,29 @@ def upload_file(self, user_id: int, filepath: str, file):
         response_json = json.loads(query_response.content)
 
         return response_json
+
+    def get_invitation_requests(self):
+        """
+        Get the list of pending connection invitations of a therapist
+
+        """
+        url = self._niceday_api_uri + 'connectionrequests'
+        query_response = self._call_api('GET', url)
+        # convert the json response into a list of dict
+        response_json = json.loads(query_response.content)
```

Comment:
I suppose you could add a check to see that the dict contains the expected keys for this request. Or does something else do this later? I suppose you still want to let through error responses, so maybe this is fine.

Reply:
Yes, this part should be transparent. But actually this verification has to be added to the api endpoint as a descriptor of the success response. I am going to do this.

Reply:
Ah yes, that would be a much better place to put it